# Oppsummering

## Brøk, desimaltall og prosent


## Prosentregning


## Vekstfaktor


## Formelregning


## Geometri


## Sannsynlighet


## Funksjoner



## Økonomi

## Brøk, desimaltall og prosent

Vi husker at brøk gjøres om til desimaltall ved å dele!

### Eksempel:

$$ \frac{3}{4} = 0,75 $$
Vi kan utføre delingen for hånd, eller med kalkulator.
Under bruker vi Python

In [1]:
3.0/4

0.75

For å gjøre dette om til prosent, ganger vi med 100:

$$ \frac{3}{4} = 0,75 = 75\% $$

Å gå fra desimaltall til brøk lar seg gjøre ganske enkelt når desimaltallet ikke fortsetter i
det uendelige:


$$ 0,684 = \frac{684}{1000} $$

Neste steg er å forkorte brøken så mye som mulig

In [2]:
684/4.0

171.0

$$ \frac{684}{1000} = \frac{684:4}{1000:4} = \frac{171}{250} $$

Siden nevneren $250 = 50\cdot 5 = 2\cdot 5^3,$ og telleren $171$ ikke er delelig på noen av disse faktorene,
er brøken forkortet så mye som mulig

## Prosentregning

### Eksempel
I et brød er det 400g sammalt hvetemel og 700g fint hvetemel.

Vi neglisjerer de andre ingrediensene

Hvor mange prosent av brødet består av sammalt hvetemel?

Før eksempelet her